lib/gitlib.sh

```diff
@@ -262,3 +262,13 @@ gitlib::repo_state () {
     return 1
   fi
 }
+
+##############################################################################
+# Returns the name of the upstream remote repository name for the local git
+# repo, e.g. "upstream" or "origin".
+#
+gitlib::upstream_remote_name() {
```

Comment:
Yeah I steal this which @ixdy mentioned here https://github.com/kubernetes/release/issues/380#issuecomment-315874982 :)